test/functionalTest/cases/1048_subscription_cache/csub_update_with_notif.test

```diff
@@ -25,16 +25,24 @@ Cache sub behaviour after update context operation
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 0 IPv4 -subCacheIval 1
+brokerStart CB 0 IPv4 -subCacheIval 2
 accumulatorStart --pretty-print
 
 --SHELL--
 
 #
-# Note we use sleep in some point to let time the DB to sync with regards to
-# cache timeSent value (WAIT_TIME = 1.5 * subCacheIval)
+# Note that we use sleep in three places to give the DB time to sync with regards to
+# the cache (WAIT_TIME = 1.5 * subCacheIval)
```

Comment:
Having changed -subCacheIval from 1 to 2 I think this formula is not still valid...

Reply:
Looking to a similar test (cases/1853_typePattern_3a/subcache_typepattern.test) in that case -subCacheIval limit was not changed. Maybe the 1 -> 2 change was done while testing but at the end is not needed and could be reverted?

Reply:
This test ...
Took me a few hours to make it work.
I have no clue why `cases/1853_typePattern_3a/subcache_typepattern.test` works without modification.
One second was simply not enough under valgrind

Reply:
But true, the formula is no longer valid

Reply:
Comment modified to reflect new reality in d47e6f5dd